restapi/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"os"
+
+	"github.com/joho/godotenv"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/OlehEngineer/goTasks/goTasks/restapi/pkg/controllers"
+	"github.com/OlehEngineer/goTasks/goTasks/restapi/pkg/service"
+	"github.com/OlehEngineer/goTasks/goTasks/restapi/pkg/usecases"
+)
+
+func init() {
```

Comment:
why need to use init?

Reply:
moved to func main()
Solved in commit cb9c63563881cecdab598e4715ca0f33420b1c56